options/components/webBottomContent/WebBottomContent.jsx

```diff
@@ -67,6 +86,32 @@ function HighlightKeyword({ url }) {
             );
           }
         })}
-    </div>
+      </div>
+    </>
+  );
+}
+
+function HighlightTitleKeyword({ url }) {
+  const { keyword } = useContext(WebSearchContext);
+
+  return (
+    <>
+      {url.title.split(keyword).map((item, index) => {
+        if (index === 0 && !item) {
+          return null;
+        } else if (index === 0 && item) {
+          return <span key={index}>{item}</span>;
+        } else if (index === 1 && item) {
+          return (
+            <span key={index}>
+              <span className="text-blue-600 font-bold px-1 inline-block mx-px">
+                {keyword}
+              </span>
+              {item}
+            </span>
+          );
+        }
+      })}
```

Comment:
keyword단위로 split 후 각 인덱스별로 하이라이팅된 keyword를 붙여주는 좋은 코드인 것 같습니다.
그러나 keyword가 여러개 일 수도 있다는 점을 까먹으신것 같습니다.
그래서 저는 다음과 같이 바꾸면 keyword가 여러개일때도 동작할 것이라고 생각합니다.
```suggestion
      {url.title.split(keyword).map((item, index) => {
        if (index === 0 && !item) {
          return null;
        } else if (index === 0 && item) {
          return <span key={index}>{item}</span>;
        } else {
          return (
            <span key={index}>
              <span className="text-blue-600 font-bold px-1 inline-block mx-px">
                {keyword}
              </span>
              {item}
            </span>
          );
        }
      })}
```